src/example-types/dateHistogram.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     search
   ) {
     let stats = _.omit(['_id'], statsAgg(value_field).$group)
-    stats.cardinality = { $addToSet: `$${value_field}` }
+    stats.cardinality = { $sum: `$${value_field}` }
```

Comment:
these changes aren't required for this PR -- they're required to fix the test that's currently failing on `master`

Reply:
I am wondering if it would not be cleaner/better to have version `0.11.3` to deal with the test issues first and then add the rest to the `0.12.0` this way later on if we need to revert for some reason we could without breaking the test cases again etc.

Reply:
@Akrion yeah, good idea. I'll close and make PRs for both